app/views/_user_util_links.html.erb

```diff
@@ -16,6 +16,7 @@
         <li><%= link_to t("hyrax.toolbar.dashboard.menu"), hyrax.dashboard_path %></li>
 
         <li class="divider"></li>
+        <li><%= link_to t("hyrax.toolbar.profile.edit_registration"), main_app.edit_user_registration_path %></li>
```

Comment:
How about I check if `:registerable` is in the devise mappings, rather than commit to new SSO-related configuration in Hyrax?